src/executor/load_executor.py

```diff
@@ -42,6 +44,7 @@ def exec(self):
         # We currently use create to empty exsiting table.
         StorageEngine.create(self.node.table_metainfo)
 
-        video_reader = OpenCVReader(self.node.file_path)
+        video_reader = OpenCVReader(
+            os.path.join('/tmp', self.node.file_path))
```

Comment:
This join '/tmp' for all load commands. I understand the reason because UPLOAD will join the '\tmp' to the user-given path, but I think this might not be a good design. One alternative can be that UPLOAD will return the generated path on the server, and the user uses the explicit path for the LOAD command?

Reply:
We had some discussions about this. Having an end-to-end system seemed to be the priority now, so we went with this design choice. With the alternative, I feel that it would not give the user the level of control over the content that he would upload to the server. For example, the user might want to create a directory structure within the server for subsequent analytics. What do you think?

Reply:
I see. I later noticed that all test cases have videos moved to /tmp, so everything works. That's good.